src/test/scala/org.openeo/OpenSearchClientTest.scala

```diff
@@ -393,4 +393,24 @@ class OpenSearchClientTest {
     // This is the product we are actually interested in.
     features.find(_.generalProperties.processingBaseline.get == processingBaseline).get
   }
+
+  @Test
+  def parseCreodiasCorruptPhoebus(): Unit = {
```

Comment:
Perhaps add a small description to this test with a link to the github issue so in the future we know why we are testing this and what 'corrupt phoebus' means.

Reply:
true! thanks